package de.l3s.boilerpipe.sax;

/**
 * Created by martynas on 20/11/14.
 */
public class Texts {
    public static final String EN1 = "A series of yeast shuttle vectors and host strains has been created to allow more efficient manipulation of DNA in Saccharomyces cerevisiae. Transplacement vectors were constructed and used to derive yeast strains containing nonreverting his3, trp1, leu2 and ura3 mutations. A set of YCp and YIp vectors (pRS series) was then made based on the backbone of the multipurpose plasmid pBLUESCRIPT. These pRS vectors are all uniform in structure and differ only in the yeast selectable marker gene used (HIS3, TRP1, LEU2 and URA3). They possess all of the attributes of pBLUESCRIPT and several yeast-specific features as well. Using a pRS vector, one can perform most standard DNA manipulations in the same plasmid that is introduced into yeast.\n";
    public static final String EN2 = "Two large-scale yeast two-hybrid screens were undertaken to identify protein–protein interactions between full-length open reading frames predicted from the Saccharomyces cerevisiae genome sequence. In one approach, we constructed a protein array of about 6,000 yeast transformants, with each transformant expressing one of the open reading frames as a fusion to an activation domain. This array was screened by a simple and automated procedure for 192 yeast proteins, with positive responses identified by their positions in the array. In a second approach, we pooled cells expressing one of about 6,000 activation domain fusions to generate a library. We used a high-throughput screening procedure to screen nearly all of the 6,000 predicted yeast proteins, expressed as Gal4 DNA-binding domain fusion proteins, against the library, and characterized positives by sequence analysis. These approaches resulted in the detection of 957 putative interactions involving 1,004 S. cerevisiae proteins. These data reveal interactions that place functionally unclassified proteins in a biological context, interactions between proteins involved in the same biological function, and interactions that link biological functions together into larger cellular processes. The results of these screens are shown here.\n";
    public static final String EN3 = "We sought to create a comprehensive catalog of yeast genes whose transcript levels vary periodically within the cell cycle. To this end, we used DNA microarrays and samples from yeast cultures synchronized by three independent methods: α factor arrest, elutriation, and arrest of a cdc15 temperature-sensitive mutant. Using periodicity and correlation algorithms, we identified 800 genes that meet an objective minimum criterion for cell cycle regulation. In separate experiments, designed to examine the effects of inducing either the G1 cyclin Cln3p or the B-type cyclin Clb2p, we found that the mRNA levels of more than half of these 800 genes respond to one or both of these cyclins. Furthermore, we analyzed our set of cell cycle–regulated genes for known and new promoter elements and show that several known elements (or variations thereof) contain information predictive of cell cycle regulation. A full description and complete data sets are available at http://cellcycle-www.stanford.edu\n";
    public static final String EN4 = "We have constructed and tested a dominant resistance module, for selection of S. cerevisiae transformants, which entirely consists of heterologous DNA. This kanMX module contains the known kanr open reading-frame of the E. coli transposon Tn903 fused to transcriptional and translational control sequences of the TEF gene of the filamentous fungus Ashbya gossypii. This hybrid module permits efficient selection of transformants resistant against geneticin (G418). We also constructed a lacZMT reporter module in which the open reading-frame of the E. coli lacZ gene (lacking the first 9 codons) is fused at its 3′ end to the S. cerevisiae ADH1 terminator. KanMX and the lacZMT module, or both modules together, were cloned in the center of a new multiple cloning sequence comprising 18 unique restriction sites flanked by Not I sites. Using the double module for constructions of in-frame substitutions of genes, only one transformation experiment is necessary to test the activity of the promotor and to search for phenotypes due to inactivation of this gene. To allow for repeated use of the G418 selection some kanMX modules are flanked by 470 bp direct repeats, promoting in vivo excision with frequencies of 10–3–10–4. The 1·4 kb kanMX module was also shown to be very useful for PCR based gene disruptions. In an experiment in which a gene disruption was done with DNA molecules carrying PCR-added terminal sequences of only 35 bases homology to each target site, all twelve tested geneticin-resistant colonies carried the correctly integrated kanMX module.\n";
    public static final String EN5 = "";
}
